packages/containers/.storybook/preview-head.html

```diff
@@ -20,4 +20,22 @@
     html.focus-outline-hidden *:focus {
         outline: none !important;
     }
+
+    .tc-side-panel > form[role="search"] {
+        margin: 0;
+        height: 33px;
+    }
+    .tc-side-panel > form > .form-group {
+        display: flex;
+        align-items: center;
+    }
+    .tc-side-panel  > form > .form-group > input {
+        margin: 0;
+        color: white;
+    }
+    .tc-side-panel > button svg.tc-svg-icon,
+    .tc-side-panel > form svg.tc-svg-icon {
+        margin: 0 10px;
+        color: white;
+    }
```

Comment:
Custom CSS !
This should go into filterbar.scss with a :global(.tc-side-panel) selector

Reply:
This is only for demo purpose. We don't need it, so adding this style in the component won't make sense. It will be integrated in the bundles and shipped with apps.